crates/bevy_text/src/text_access.rs

```diff
@@ -281,15 +303,15 @@ impl<'w, 's, R: TextRoot> TextWriter<'w, 's, R> {
     /// Gets the [`TextStyle`] of a text span within a text block at a specific index in the flattened span list.
```

Comment:
Doc links are broken here.

Reply:
Yeah going to take me a while to work through all of these 😅